lib/src/main/kotlin/com/swmansion/starknet/account/Account.kt

```diff
@@ -19,10 +19,11 @@ interface Account {
      *
      * @param call a call to be signed
      * @param params additional execution parameters for the transaction
+     * @param forFeeEstimate parameter indicating whether the sign will be only used for cost estimation
```

Comment:
```suggestion
     * @param forFeeEstimate setting it to `true` changes the signed transaction version so it can only be used for fee estimation
```

Reply:
I wonder if we should add a note somewhere to explain what version should be used in case someone is implementing the Account interface themselves.

Reply:
```
* @param forFeeEstimate when set to 'true', it changes the version to `2^128+version` so the signed transaction can only be used for fee estimation
```
🤔 ?

Reply:
Sounds ok!